dev-support/hbase-personality.sh

```diff
@@ -597,8 +618,9 @@ function hadoopcheck_rebuild
     fi
   else
     yetus_info "Setting Hadoop 3 versions to test based on branch-2.6+/master/feature branch rules"
+    # Isn't runnung these tests with the default Hadoop version redundant ?
     if [[ "${QUICK_HADOOPCHECK}" == "true" ]]; then
-      hbase_hadoop3_versions="3.3.6 3.4.0"
```

Comment:
You can see the pattern, actually we will use the latest one for a minor release line for quick hadoop check...

Reply:
Done